net-misc/openssh-contrib/openssh-contrib-9.3_p1.ebuild

```diff
@@ -210,13 +210,13 @@ src_prepare() {
 	fi
 
 	if use hpn ; then
-		local hpn_patchdir="${T}/${P}-hpn${HPN_VER}"
+		local hpn_patchdir="${T}/openssh-${PV}-hpn${HPN_VER}"
 		mkdir "${hpn_patchdir}" || die
 		cp $(printf -- "${DISTDIR}/%s\n" "${HPN_PATCHES[@]}") "${hpn_patchdir}" || die
 		pushd "${hpn_patchdir}" &>/dev/null || die
 		eapply "${WORKDIR}/${HPN_GLUE_PATCH}"
 		use X509 && eapply "${WORKDIR}/${X509_HPN_GLUE_PATCH}"
-		use sctp && eapply "${FILESDIR}"/${PN}-8.5_p1-hpn-${HPN_VER}-sctp-glue.patch
+		use sctp && eapply "${FILESDIR}"/openssh-8.5_p1-hpn-${HPN_VER}-sctp-glue.patch
 		popd &>/dev/null || die
 
 		eapply "${hpn_patchdir}"
```

Comment:
Below, for HPN_DISABLE_MTAES, please link to https://bugs.gentoo.org/354113#c6 so finally 12 years after someone pointed it out, the comment for the test case makes it into the ebuild